Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_GCP_Enrichment.yml

```diff
@@ -1120,138 +1094,118 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "32":
-    id: "32"
-    taskid: 6326d4c8-637a-4734-8134-808b2bcbb99b
-    type: regular
-    task:
-      id: 6326d4c8-637a-4734-8134-808b2bcbb99b
-      version: -1
-      name: Set tags grid field
-      description: |-
-        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. For example:
-        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
-      scriptName: GridFieldSetup
-      type: regular
-      iscommand: false
-      brand: Builtin
+  "30":
+    id: "30"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
     nexttasks:
       '#none#':
       - "9"
+    note: false
+    quietmode: 0
     scriptarguments:
       gridfield:
-        simple: asmtags
+        simple: asmserviceowner
       keys:
-        simple: Key,Value,Source
+        simple: Name,Email,Source,Timestamp
       val1:
-        complex:
-          root: GCPIAM.TagBindings
-          accessor: key
+        simple: n/a
       val2:
         complex:
-          root: GCPIAM.TagBindings
-          accessor: value
+          accessor: serviceAccounts
+          root: GoogleCloudCompute.Instances
+          transformers:
+          - operator: FirstArrayElement
+          - args:
+              field:
+                value:
+                  simple: email
+            operator: getField
       val3:
-        simple: GCP-project-folder-tags
+        simple: Service account on instance
+      val4:
+        complex:
+          root: TimeNowUnix
+          transformers:
+          - operator: TimeStampToDate
     separatecontext: false
-    continueonerrortype: ""
-    view: |-
-      {
-        "position": {
-          "x": 2230,
-          "y": 1265
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
     skipunavailable: false
-    quietmode: 0
-    isoversize: false
-    isautoswitchedtoquietmode: false
-  "34":
-    id: "34"
-    taskid: 1414247b-89e4-4a4d-88da-ca5ed991128d
-    type: title
     task:
-      id: 1414247b-89e4-4a4d-88da-ca5ed991128d
-      version: -1
-      name: Tags - Project/Folder/Organization Tags
```

Comment:
Why is this being removed? This was just added by #25993

Reply:
i think it might just be getting re-arranged -- see lines 1158-1168

Reply:
Looks good to me 👍🏽